src/models/users/useGetUsers.js

```diff
@@ -1,16 +1,9 @@
 import queryKeys from '../../constants/queryKeys';
 import useFetch from '../../hooks/useFetch';
 
-const limit = 20;
-const offset = 0;
-
 export default function useGetUsers() {
   return useFetch({
     queryKey: queryKeys.users,
     url: '/users/',
-    data: {
-      limit,
-      offset,
-    },
```

Comment:
I don't think that `limit` and `offset` should be enforced without a way to programmatically change them. We don't seem to be trying to anywhere, but, where we use this, we also don't seem to expect the response to be limited to a subset of users.

Reply:
Yeah... this has been giving me a "cruft" vibe for a minute. Glad to see it go.